src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier{
+    public int getRandomBall(){
+        Random random = new Random();
+        return random.nextInt(101) ;
+    }
+
+    @Override
+    public String getRandomColor() {
+        return super.getRandomColor();
+    }
```

Comment:
This method override is redundant since it only calls the superclass method without any additional logic. You can remove this override.